README.md

```diff
@@ -302,3 +289,74 @@ AWS uses two virtualization types to launch the EC2 instances: Para-Virtualizati
 ***RELATED INFORMATION***
 
 *You can download the white paper "Amazon EC2 Deployment Guide for Hazelcast IMDG" [here](https://hazelcast.com/resources/amazon-ec2-deployment-guide/).*
+
+## Configuration for AWS ECS
+
+Hazelcast member discover on both ECS Fargate and ECS EC2 is supported. The ECS mode is activated by setting `host-header` to an ECS endpoint address, typically `ecs.amazonaws.com`.
+Most of the EC2 configuration above applies to ECS mode too.
+
+#### XML Configuration
+
+```xml
+<hazelcast>
+  <network>
+    <join>
+      <multicast enabled="false"/>
+      <aws enabled="true">
+        <host-header>ecs.amazonaws.com</host-header>
+        <region>us-west-1</region>
+        <access-key>my-access-key</access-key>
+        <secret-key>my-secret-key</secret-key>
+        <hz-port>5701-5708</hz-port>
+      </aws>
+    </join>
+  </network>
+</hazelcast>
+```
+
+#### Java-based Configuration
+
+```java
+config.getNetworkConfig().getJoin().getMulticastConfig().setEnabled(false);
+config.getNetworkConfig().getJoin().getAwsConfig().setEnabled(true)
+      .setProperty("host-header", "ecs.amazonaws.com")
+      .setProperty("region", "us-west-1")
+      .setProperty("access-key", "my-access-key")
+      .setProperty("secret-key", "my-secret-key")
+      .setProperty("hz-port", "5701-5708");
+```
+
+Here are the definitions of the properties
+
+* `host-header`: URL that is the entry point for a web service; must start with "`ecs.`" for ECS support.
+* `region`: region where Hazelcast members are running; if not set, `us-east-1` region is used.
+* `access-key`, `secret-key`: access and secret keys of your account on EC2; if not set, `iam-role` is used.
+* `iam-role`: AWS IAM Role to fetch credentials (used if `access-key`/`secret-key` not specified);
+  if not set, the default IAM Role assigned to ECS Task is used.
+* `connection-timeout-seconds`: maximum amount of time Hazelcast will try to connect to a well known member before giving up;
+  setting this value too low could mean that a member is not able to connect to a cluster; setting the value too high means that member startup could slow down because of longer timeouts (for example, when a well known member is not up); its default value is 5.
+* `hz-port`: a range of ports where the plugin looks for Hazelcast members; if not set, the default value `5701-5708` is used.
```

Comment:
I agree, this would be a breaking change for EC2 users though, so we should be careful and highlight this fact in the docs.

Reply:
No, what I meant is that for EC2 the `5701-5708` is the correct default value. It makes sense to have multiple Hazelcast memebers on one EC2 machine. However, I think for ECS (Fargate), the default value should be `5701`, because it doesn't make sense to have multiple Hazelcast members in one task (container).